packages/core/pattern.mjs

```diff
@@ -1376,7 +1376,7 @@ export function polymeter(...args) {
 
 // alias
 export function pm(...args) {
```

Comment:
Aha thanks for the fix!

While you're here, it would probably be better to just have `export pm = polymeter;` instead of the function

Reply:
I had tried that before, but it causes an error at startup:
```
pattern.mjs?t=1679428469545:1142 Uncaught (in promise) ReferenceError: Cannot access 'pm' before initialization
    at pattern.mjs?t=1679428469545:1142:3
```
This error stems from this code:

```
// methods that create patterns, which are added to patternified Pattern methods
// TODO: remove? this is only used in old transpiler (shapeshifter)
Pattern.prototype.factories = {
  pure,
  stack,
  slowcat,
  fastcat,
  cat,
  timeCat,
  sequence,
  seq,
  polymeter,
  pm,
  polyrhythm,
  pr,
};
// the magic happens in Pattern constructor. Keeping this in prototype enables adding methods from the outside (e.g. see tonal.ts)

```

So I shifted `export pm = polymeter;` before that `Pattern.prototype.factories` statement.